config/core/resources/cloudauditlogssource.yaml

```diff
@@ -194,9 +194,9 @@ spec:
                 description: >
                   ID of the Stackdriver sink used to publish audit log messages.
   - <<: *version
+    # TODO: remove v1alpha1 schema from CRD
     name: v1alpha1
-    # TODO: Flip served bit of v1alpha1 in https://github.com/google/knative-gcp/issues/1544.
-    served: true
+    served: false
```

Comment:
I checked knative-eventing. It looks like we just deleted v1alpha1 version in crd definition.
Maybe create another issue to clean all v1alpha1 version in crds.